pkgs/development/libraries/cmark-gfm/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+
+stdenv.mkDerivation rec {
+  pname = "cmark-gfm";
+  version = "0.29.0.gfm.0";
+
+  src = fetchFromGitHub {
+    owner = "github";
+    repo = "cmark-gfm";
+    rev = version;
+    sha256 = "0wfr3xwl4wria8vld71flv6vpsdj9aj81yqvj0azidyb8p229a1l";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  doCheck = false;
```

Comment:
Can you comment why tests are disabled?

Reply:
heya, see PR description - I had troubles running tests as they load the library dynamically, not sure how to patch them so they would find and could load the library when compiled. As I tested the resulting library and binary manually I was thinking we could have the tests disabled for now as this is a new package and then I could revisit adding tests in another PR